MediaBrowser.Model/MediaInfo/LiveStreamRequest.cs

```diff
@@ -22,12 +30,7 @@ public class LiveStreamRequest
         public bool EnableDirectStream { get; set; }
         public MediaProtocol[] DirectPlayProtocols { get; set; }
 
-        public LiveStreamRequest()
-        {
-            EnableDirectPlay = true;
-            EnableDirectStream = true;
-            DirectPlayProtocols = new MediaProtocol[] { MediaProtocol.Http };
-        }
+
```

Comment:
do we really need 3 empty lines, or is it github showing borked diff?

Reply:
I fixed it